core/src/main/java/org/elasticsearch/monitor/os/OsProbe.java

```diff
@@ -191,6 +191,11 @@ private String readSingleLine(final Path path) throws IOException {
     // pattern for lines in /proc/self/cgroup
     private static final Pattern CONTROL_GROUP_PATTERN = Pattern.compile("\\d+:([^:,]+(?:,[^:,]+)?):(/.*)");
 
+    // this property is to support a hack to workaround an issue with Docker containers not mounting the cgroups hierarchy inconsistently
+    // with respect to /proc/self/cgroup; for Docker containers this should be set to "/"
+    private static final String CONTROL_GROUPS_PATH_OVERRRIDE =
+        (String) BootstrapInfo.getSystemProperties().get("es.cgroups.path.override");
```

Comment:
This can just be `System.getProperty`? I thought the point of `BootstrapInfo.getSystemProperties()` was when you wanted to call `System.getProperties()` (which we have in forbidden apis)